python-frontend/src/main/java/org/sonar/python/tree/NumericLiteralImpl.java

```diff
@@ -57,6 +57,9 @@ public long valueAsLong() {
     if (literalValue.startsWith("0b") || literalValue.startsWith("0B")) {
       return Integer.valueOf(literalValue.substring(2), 2);
     }
+    if (literalValue.startsWith("0o") || literalValue.startsWith("0O")) {
```

Comment:
as discussed, Python 2 octal literals are prefixed with `0`. 
This means we might have an FP with the following code:
```python
os.chmod("foo", 0750)
```

Reply:
Good catch! Fixed in d4b8d61bd603df59b4190b95794446f1cc0a362e.